pkg/addons/embeddedclusteroperator/embeddedclusteroperator.go

```diff
@@ -138,7 +143,15 @@ func (e *EmbeddedClusterOperator) GenerateHelmConfig(onlyDefaults bool) ([]v1bet
 }
 
 func (e *EmbeddedClusterOperator) GetAdditionalImages() []string {
-	return []string{UtilsImage}
+	for image, tag := range Metadata.Images {
```

Comment:
Do you really need a loop here since this is a map?

Reply:
No, that is not necessary. Solved.